analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/SymbolicExecution/SETestContext.cs

```diff
@@ -66,6 +70,28 @@ public void Main(bool boolParameter{additionalParameters})
             return new(code, AnalyzerLanguage.CSharp, additionalChecks);
         }
 
+        public static SETestContext CreateCSForLocalFunction(string methodBody, string additionalParameters, string localFunctionName, params SymbolicCheck[] additionalChecks)
+        {
+            var code = $@"
+using System.Collections.Generic;
+
+public class Sample
+{{
+    public static int StaticField;
+    public static int StaticProperty {{get; set;}}
+    public int Property {{get; set;}}
+    private int field;
+
+    public void Main(bool boolParameter{additionalParameters})
+    {{
+        {methodBody}
+    }}
+
+    private void Tag(string name, object arg = null) {{ }}
+}}";
+            return new(code, AnalyzerLanguage.CSharp, additionalChecks, localFunctionName);
+        }
+
```

Comment:
IT is used in the `Execute_CapturedVariable_NotCleared` test.

Reply:
Hmm, VS didn't show me that reference. 
So it can be renamed to `CreateCS` overload. The duplication can be removed and one `CreateCS` (without the extra argument) can just call the other `CreateCS` with extra argument.